src/ng/compile.js

```diff
@@ -2555,10 +2574,10 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
             break;
 
           case '=':
-            if (optional && !attrs[attrName]) {
-              return;
-            }
             parentGet = $parse(attrs[attrName]);
+
+            if (parentGet === noop && optional) break;
```

Comment:
Does this behaves exactly as the previous code? If this is the case, please keep the code as it was before